src/markdown.test.js

```diff
@@ -46,7 +47,41 @@ describe('reportToComment', () => {
 
         expect(reportToComment({report})).toMatchInlineSnapshot(`
             "
-            Readability after merging this PR:
+            Readability after merging this PR: 1/100 (🟢 +1)
+
+            Path | Readability
```

Comment:
suggestion: maybe this would be more understandable for a less technically minded audience 
```suggestion
            File | Readability
```